File: fax/v1/fax/create-example-2/create-example-2.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$fax = $twilio->fax->v1->faxes
                       ->create("sip:kate@example.com",
                                "https://www.twilio.com/docs/documents/25/justthefaxmaam.pdf",
                                array(
                                    'from' => "Jack",
                                    'sipAuthPassword' => "secret",
                                    'sipAuthUsername' => "jack"
                                )
                       );

print($fax.sid);

File: api/v2010/call/create-example-2/create-example-2.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

call = @client.calls.create(
                       url: 'http://demo.twilio.com/docs/voice.xml',
                       to: 'client:charlie',
                       from: '+15017122661'
                     )

puts call.sid
